common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -345,28 +360,20 @@ def answer_problem(self, correct):
             self.problem_page.click_choice("choice_1")
 
     @attr('shard_7')
-    def test_can_show_hide_answer(self):
+    def test_can_show_answer(self):
```

Comment:
Yes. This test is from legacy code, would override the one above, but need the problem meta focus here.

All tests passed. I will go ahead and merge this. :)